modules/openapi-generator/src/test/resources/bugs/issue_11464.yaml

```diff
@@ -0,0 +1,32 @@
+openapi: '3.0.3'
+info:
+  version: 1.0.0
+  title: Example Api
+paths:
+  /single:
+    get:
+      summary: Single Tag
+      operationId: get_single_tag
+      tags:
+        - tag1
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+  /multible:
```

Comment:
Might be good to fix the typo - `multiple` - so that any future reader doesn't need to be unsure of the typo